ephios/core/templates/core/fragments/event_list_list_mode.html

```diff
@@ -53,21 +58,21 @@
                             </div>
                         {% endif %}
                     </div>
-                    <a class="w-100 py-2 text-reset event-list-item-link"
-                       href="{{ event.get_absolute_url }}">
+                    <div class="w-100 py-2 event-list-item">
                         <div class="grid-wrapper m-0 py-0 ps-2 pe-3">
-                            <div class="grid-title">
+                            <a class="grid-title event-list-item-link" href="{{ event.get_absolute_url }}">
```

Comment:
is there a reason why there are multiple links instead of the event-list-item being the link?

Reply:
I didn't want the action are to be a link in case we want to put other links there at some point. Same as github notifications.